app/javascript/components/molecules/FormGrid.js

```diff
@@ -25,11 +25,13 @@ FormGrid.Row.defaultProps = {
 }
 
 // NOTE: Grid is hardcoded. This should not be generic element
-FormGrid.Item = ({ children }) => (
-  <div className='form-grid_item small-12 medium-6 large-3 column'>
-    {children}
-  </div>
-)
+FormGrid.Item = ({ children, wide }) => {
+  return (
+    <div className={`form-grid_item small-12 medium-6 ${wide ? 'large-6' : 'large-3'} column`}>
```

Comment:
nit: for long lists of classnames with conditional values like this I think it's preferrable to use the classnames util:

```
import classNames from 'classnames'
classNames(
  'form-grid_item',
  'small-12',
  'medium-6',
  'column',
  { 'large-6': wide, 'large-3': !wide },
  { wide ? 'large-6' : 'large-3' } // another option that does the same thing
)
```

Reply:
what is this change for? Can we pull it out and I can fix the layout in the next PR if necessary?

Reply:
`wide` option is to spread grid columns to the full width of the container as it is in Figma designs.